src/components/ItaliaTheme/View/CartellaModulisticaView/DocRow.jsx

```diff
@@ -77,7 +77,7 @@ const DocRow = ({ doc, items }) => {
       key={doc['@id']}
     >
       {/*Only title and/or description, no files */}
-      {(!items || items.length == 0) && (
+      {(!items || items.length === 0) && (
```

Comment:
So as the Array.length property is described by js specs is it not enough to use just `!items.length` ?

Reply:
correct, but, IMHO, less intelligible